index.js

```diff
@@ -5,18 +5,14 @@
  */
 
 module.exports = (function() {
-  var version = process.version;
-
-  if (version.indexOf('v') > -1) {
-    version = version.split('v')[1];
-  }
+  var version = process.versions.node;
 
   var split = version.split('.');
 
   return {
-    original: process.version,
+    original: 'v' + version,
```

Comment:
Is this an over-optimization?

Reply:
or maybe keep `process.version` here ?

Reply:
Your call, if you want, I can revert that.